Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Message.hs

```diff
@@ -119,8 +122,10 @@ fetchMedia (driverId, _) filePath = do
 messageSeen :: (Id SP.Person, Id DM.Merchant) -> Id Domain.Message -> Flow APISuccess
 messageSeen (driverId, _) messageId = do
   _ <- Esq.runInReplica $ QP.findById driverId >>= fromMaybeM (PersonNotFound driverId.getId)
-  Esq.runTransaction $ MRQ.updateSeenAndReplyByMessageIdAndDriverId messageId (cast driverId) True Nothing
-
+  messageDetails <- Esq.runInReplica $ MRQ.findByMessageIdAndDriverId messageId (cast driverId) >>= fromMaybeM (InvalidRequest "Message not found")
```

Comment:
why this query?

Reply:
to fetch weather the message is already seen , if it is seen  then i will not increase the message count otherwise i will increase it